aws-lambda-layer/Dockerfile

```diff
@@ -1,14 +1,17 @@
-FROM lambci/lambda:build-ruby2.5
+FROM amazonlinux:2
 
 ARG FREETDS_VERSION=1.1.6
 
 ENV INSTALLDIR='/tmp/freetds'
 ENV FREETDS_VERSION=$FREETDS_VERSION
 
 RUN yum update -y
-RUN yum install wget build-essential libc6-dev -y
+RUN yum install wget tar gzip zip gcc make -y
 
-RUN mkdir $INSTALLDIR
+RUN mkdir $INSTALLDIR \
+    build \
+    '/var/task' \
+    '/var/task/build'
 
 RUN wget https://github.com/veracross/freetds/raw/master/freetds/freetds-${FREETDS_VERSION}.tar.gz && \
```

Comment:
In a future iteration of this project, we may want to split out the download step from the build step. However, that doesn't need to be done in this PR